src/components/Asset/AssetActions/AssetStats/index.tsx

```diff
@@ -5,17 +5,25 @@ import styles from './index.module.css'
 export default function AssetStats() {
   const { asset } = useAsset()
 
+  // TODO: get this number for each asset
```

Comment:
another hardcoded value we need to fetch, to keep stats on teaser and asset details consistent

Reply:
didn't see you added it here as well, will implement it here